### PR TITLE
Add getEstimateTranslation utility function

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What is the purpose of this pull request?

Adds the `getEstimateTranslator` utility function, that returns a string of the 
estimate translated.

#### What problem is this solving?

This is useful for scenarios where you need to handle a string instead of a 
React component, such as using the estimate in a `<options>` children, which 
must be a string.

#### How should this be manually tested?

N/A

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/TranslateEstimate.test.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+
```
Comment (wdsrocha) [inline react/TranslateEstimate.test.js:2]: (related to the whole file)
tests were failing?

**Review by wdsrocha — COMMENTED** [review]
Besides improving the code quality (ty 🙏🏾), the only "real change" here is that `TranslateEstimate` now is just a wrapper for `getEstimateTranslation` so it can be called as a component and as a function, right?

File: react/TranslateEstimate.test.js (hunk 1 above)
Comment (lucasecdb) [reply react/TranslateEstimate.test.js:2]: yep, we changed the translations but didn't update the tests accordingly

**Comment by lucasecdb** [comment]
> Besides improving the code quality (ty 🙏🏾), the only "real change" here is that `TranslateEstimate` now is just a wrapper for `getEstimateTranslation` so it can be called as a component and as a function, right?

exactly!

**Review by wdsrocha — APPROVED** [review]

**Review by jeffersontuc — APPROVED** [review]

**Pull request merged**